### ISSUE TITLE
Why the accuracy of pretrained resnet50 model is very low

### ISSUE THREAD

**Issue by AAA-Fan** [Cadene/pretrained-models.pytorch#199]
Test: [2570/5000]       Time 0.024 (0.060)      Loss 19.2762 (18.3547)  Acc@1 0.000 (0.117)    Acc@5 0.000 (0.381)

This is the recognition result on test dataset of imagenet.

**Comment by ManoharSai2000** [comment]
I have been using the same pretrained model and this is either a wrong model or an incomplete one.
You can download this here.
https://github.com/rwightman/pytorch-image-models/releases/download/v0.1-weights/seresnet50_ra_224-8efdb4bb.pth
